stable/atlantis/templates/_helpers.tpl

```diff
@@ -0,0 +1,32 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "atlantis.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "atlantis.fullname" -}}
+{{- if .Values.fullnameOverride -}}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- if contains $name .Release.Name -}}
+{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
```

Comment:
also add https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md and apply that serviceaccount for the deployment/statefulset.

Reply:
@lkysow @davidkarlsen I'm fairly new to (i.e. ignorant of) RBAC resources, so I'd appreciate a sanity check of my understanding:

As far as I can tell, the only thing that would need to (optionally) be created is a `ServiceAccount` with no permissions because Atlantis does not perform any actions against the k8s API.

This is what I did in the latest push.

Reply:
can you add documentation for the new fields in the readme too?

Reply:
Done.